<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mealprep</class>
 <widget class="QMainWindow" name="mealprep">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mealprep</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>521</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="pantry_tab">
     <attribute name="title">
      <string>Pantry</string>
     </attribute>
     <widget class="QFrame" name="add_ingredients_frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>731</width>
        <height>101</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>711</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="add_ingredients_layout">
        <item>
         <widget class="QLabel" name="ingredient_label">
          <property name="text">
           <string>Ingredient: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ingredient_line_edit"/>
        </item>
        <item>
         <widget class="QLabel" name="quantity">
          <property name="text">
           <string>Quantity: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="quantity_spin_box"/>
        </item>
        <item>
         <widget class="QLabel" name="quantity_label">
          <property name="text">
           <string>Unit: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="unit_combo_box">
          <item>
           <property name="text">
            <string>pcs</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cup</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>tbsp</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>bowl</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="add_button">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLabel" name="add_ingredients_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Ingredients</string>
      </property>
     </widget>
     <widget class="QFrame" name="pantry_frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>210</y>
        <width>731</width>
        <height>271</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QTableWidget" name="pantry_table">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>711</width>
         <height>211</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="get_cookable_recipe_button">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>171</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Get Cookable Recipes</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>180</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Pantry</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="recipe_tab">
     <attribute name="title">
      <string>Recipe</string>
     </attribute>
     <widget class="QLabel" name="cookable_recipe_lable">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Cookable Recipes</string>
      </property>
     </widget>
     <widget class="QFrame" name="recipe_frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>721</width>
        <height>411</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QListWidget" name="cookable_recipe_list_view">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="cookable_recipe_ingredients_label">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ingredients for selected recipe</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="cookable_recipe_ingredients_table">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>451</width>
         <height>301</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>360</y>
         <width>431</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Cook Selected Recipe</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Recipe Book</string>
     </attribute>
     <widget class="QFrame" name="recipe_book_frame">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>721</width>
        <height>411</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Raised</enum>
      </property>
      <widget class="QListWidget" name="recipe_book_list_view">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>211</width>
         <height>381</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="recipe_book_ingredients_label">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>20</y>
         <width>191</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Ingredients for selected recipe:</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="recipe_book_ingredients_table">
       <property name="geometry">
        <rect>
         <x>250</x>
         <y>50</y>
         <width>451</width>
         <height>341</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="recipe_book_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Recipes</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
